doc/adding_lints.md

```diff
@@ -526,6 +527,81 @@ Before submitting your PR make sure you followed all of the basic requirements:
 - \[ ] Added lint documentation
 - \[ ] Run `cargo dev fmt`
 
+## Adding configuration to a lint
+
+Clippy supports the configuration of lints values using a `clippy.toml` file in the workspace 
+directory. Adding a configuration to a lint can be useful for thresholds or to constrain some
+behavior that can be seen as a false positive for some users. Adding a configuration is done 
+in the following steps:
+
+1. Adding a new configuration entry to [clippy_lints::utils::conf](/clippy_lints/src/utils/conf.rs)
+    like this:
+    ```rust
+    /// Lint: LINE_NAME. <The configuration field doc comment>
```

Comment:
What does `LINE_NAME` signify here?

Reply:
The `LINT_NAME` stands for the lint that the configuration is linked to. The `LINT_NAME` identifier is also later used for the lint. Do you think I should clarify it? :upside_down_face:

Reply:
Ah, then you should probably just rename it from `LINE_NAME` to `LINT_NAME`.

Reply:
Oh, well reading is not always my strong suite :sweat_smile: I'll update it right away :upside_down_face: